android/app/src/main/java/io/mosip/registration_client/api_services/DynamicDetailsApi.java

```diff
@@ -81,20 +84,20 @@ public void getDocumentValues(@NonNull String categoryCode, String applicantType
     @Override
     public void getLocationValuesBasedOnParent(@Nullable String parentCode, @NonNull String hierarchyLevelName, @NonNull String langCode, @NonNull DynamicResponsePigeon.Result<List<DynamicResponsePigeon.GenericData>> result) {
         List<DynamicResponsePigeon.GenericData> locationList = new ArrayList<>();
+        List<GenericValueDto> genericValueList = new ArrayList<>();
         try {
-            int hierarchyLevel = this.masterDataService.getHierarchyLevel(hierarchyLevelName);
-            List<GenericValueDto> genericValueList = this.masterDataService.findLocationByParentHierarchyCode(parentCode, langCode);
+            genericValueList = this.masterDataService.findLocationByParentHierarchyCode(parentCode, langCode);
             genericValueList.forEach((v) -> {
                 DynamicResponsePigeon.GenericData location = new DynamicResponsePigeon.GenericData.Builder()
                         .setCode(v.getCode())
                         .setName(v.getName())
                         .setLangCode(v.getLangCode())
-                        .setHierarchyLevel((long) hierarchyLevel)
+                        .setHierarchyLevel((long) 0)
```

Comment:
we should add explanation on why hierarchy level is changed to 0

Reply:
Removed from here as it is not being used now.